cmd/provider/main.go

```diff
@@ -61,8 +62,12 @@ func main() {
 	})
 	kingpin.FatalIfError(err, "Cannot create controller manager")
 
+	for gv, kinds := range apis.Schemas {
+		mgr.GetScheme().AddKnownTypes(gv, kinds...)
+		metav1.AddToGroupVersion(mgr.GetScheme(), gv)
+	}
```

Comment:
@muvaf are you planning to update existing providers with this pattern as well?

Reply:
Yes, I'm planning to take a pass after @turkenh finalizes doing the same thing for controller registration and updates here. We'll be able to remove yet another manual step from ACK as well.